pipelines/devops-agent-deploy.yaml

```diff
@@ -51,78 +51,7 @@ pool:
   vmImage: ubuntu-22.04  # Ubuntu-latest by default does not have Terraform installed
 
 stages:
-  - stage: Plan

```

Comment:
We're planning to eventually migrate this to its own repo, but i think the infra deployment is needed since the main infra deployment doesn't deploy this pat of the infra